gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -25,11 +25,42 @@ import { Color3 } from "babylonjs/Maths/math.color";
 import { WebRequest } from "babylonjs/Misc/webRequest";
 import { IPointerEvent, IWheelEvent } from "babylonjs/Events/deviceInputEvents";
 
+declare var GUIEDITOR: any;
+declare var BABYLON: any;
+
+/**
+ * Interface used to configure the node material editor
+ */
+ export interface IGUIEditorOptions {
+    /** Define the URl to load node editor script */
+    editorURL?: string;
+}
+
 /**
  * Class used to create texture to support 2D GUI elements
  * @see https://doc.babylonjs.com/how_to/gui
  */
 export class AdvancedDynamicTexture extends DynamicTexture {
+    /** Define the Url to load node editor script */
+    public static EditorURL = `https://unpkg.com/babylonjs-gui-editor@${Engine.Version}/babylon.guiEditor.js`;
```

Comment:
the same comment about umd vs. es6 - this is a UMD version of the editor that might not work well if the gui package is es6